### PR TITLE
Added support for SteelSeries HP Omen keyboard

### PR DESCRIPTION
**Pull request by kybl**
Hi! I bought HP Omen keyboard by SteelSeries (https://support.hp.com/us-en/product/omen-by-hp-keyboard-with-steelseries/12733358). You can search it also by product code X7Z97AA. It looks like a clone of Apex 350, it is identified as `SteelSeries SteelSeries Apex 350 HP Omen` and `lsusb` says it has code `1038:120a`.

I found your code very useful, however, there is not support for my keyboard. This PR adds `1038:120a` as a supported device. With this patch, everything works as expected for me.

Also, I fixed one typo.

### CODE REVIEW

File: 90-apex.hwdb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # 3) sudo udevadm control --reload
 
 #keyboard:usb:v1038p120[02]*
-evdev:input:b0003v1038p120[0268]*
+evdev:input:b0003v1038p120[0268A]*
```
Comment (kybl) [inline 90-apex.hwdb:6]: Please note that on this line, there is missing `1600` for keyboard Apex M800. Is this intended? I needed to put my product ID here in order to be able to redefine default keys (and without correct code, the whole file is pointless).
Maybe there should be another PR for it.

**Comment by Gibtnix** [comment]
Thanks, looks good to me :+1: 
Regarding your question referring to the Apex M800 - actually I did not test it personally with an M800 since I am not equipped with one. The M800 support ist based on someone who offered testing support (there is also an issue for that) and at least as far as I know, all six macro keys already work fine without the hwdb file. Still if there are modifications required as well, I appreciate any respective PRs.

**Comment by kybl** [comment]
Thanks for merging :-)

Ad Apex M800: It seems to be without most of the keys from other keyboards, so it is good as it is. If not, someone else with actual keyboard can do that.

Thanks for your work!

**Comment by Gibtnix** [comment]
You're welcome and thanks as well! :-)
